moviePage.html

```diff
@@ -0,0 +1,21 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="styles/global.css">
+    <link rel="stylesheet" href="styles/moviePage.css">
+    <link rel="apple-touch-icon" sizes="180x180" href="./icons/favicon/apple-touch-icon.png">
+    <link rel="icon" type="image/png" sizes="32x32" href="./icons/favicon/favicon-32x32.png">
+    <link rel="icon" type="image/png" sizes="16x16" href="./icons/favicon/favicon-16x16.png">
+    <link rel="icon" type="icon" href="./icons/favicon/favicon.ico">
+    <title> </title>
+</head>
+<body>
+    <Section id="movie__info">
+    </Section>
+    <script async src="./js/moviePage.js" type="module"></script>
+    <script async src="./js/constants.js" type="module"></script>
```

Comment:
I'm curious why the constants are loaded only the second

Reply:
fixed